modules/openxr/openxr_interface.cpp

```diff
@@ -154,9 +155,14 @@ PackedStringArray OpenXRInterface::get_suggested_tracker_names() const {
 	// These are hardcoded in OpenXR, note that they will only be available if added to our action map
 
 	PackedStringArray arr = {
-		"left_hand", // /user/hand/left is mapped to our defaults
-		"right_hand", // /user/hand/right is mapped to our defaults
-		"/user/treadmill",
+		"head", // XRPositionalTracker for the users head (Mapped f
+		"left_hand", // XRControllerTracker for the users left hand (Mapped from OpenXR /user/hand/left)
+		"right_hand", // XRControllerTracker for the users right hand (Mapped from OpenXR /user/hand/right)
+		"/user/hand_tracker/left", // XRHandTracker for the users left hand
+		"/user/hand_tracker/right", // XRHandTracker for the users right hand
+		"/user/body_tracker", // XRBodyTracker for the users body
+		"/user/face_tracker", // XRFaceTracker for the users face
+		"/user/treadmill"
```

Comment:
It's officially part of the OpenXR spec, we just don't do anything with it. In theory it would be possible to enable it through an extension in OpenXR.

Reply:
I left it there, but there's very little information in the spec about it.